src/eei.cpp

```diff
@@ -81,6 +95,14 @@ Literal EthereumInterface::callImport(Import *import, LiteralList& arguments) {
       return Literal();
     }
 
+    if (import->base == Name("getCodeSize")) {
+      cout << "codesize\n";
+
+      size_t code_size = context->fn_table->get_code(NULL, context, &msg.address);
```

Comment:
This code is nice, but unfortunately it will be used in `getExternalCodeSize`. For `getCodeSize` we have it locally as `code.size()`.